clients/v1_repl/v1_repl.go

```diff
@@ -0,0 +1,912 @@
+		return prompt.FilterFuzzy(s, d.GetWordBeforeCursor(), true)
+	} else if strings.HasPrefix(currentLineUpper, "USE ") {
+		for _, db := range c.Databases {
+			s = append(s, prompt.Suggest{Text: "\"" + db + "\"", Description: "Table Name"})
+		}
+		return prompt.FilterFuzzy(s, d.GetWordBeforeCursor(), true)
+	} else if strings.HasPrefix(currentLineUpper, "SELECT ") {
+		if isMatch, _ := regexp.Match(`FROM `+identRegex("from_clause")+`?$`, []byte(currentLineUpper)); isMatch {
+			if c.Database != "" && c.RetentionPolicy != "" {
+				for _, m := range c.Measurements {
+					s = append(s, prompt.Suggest{Text: "\"" + m + "\"", Description: fmt.Sprintf("Measurement on \"%s\".\"%s\"", c.Database, c.RetentionPolicy)})
+				}
+			}
+			if c.Database != "" {
+				for _, rp := range c.RetentionPolicies {
+					s = append(s, prompt.Suggest{Text: "\"" + rp + "\"", Description: "Retention Policy on " + c.Database})
+				}
+			}
+			for _, db := range c.Databases {
+				s = append(s, prompt.Suggest{Text: "\"" + db + "\"", Description: "Table Name"})
+			}
+			return prompt.FilterFuzzy(s, d.GetWordBeforeCursorUntilAnySeparator(" ", "."), true)
+		}
+	}
+	return append(
+		prompt.FilterHasPrefix(replKeywords, strings.ToLower(d.CurrentLine()), false),
+		prompt.FilterHasPrefix(allInfluxQLKeywords, strings.ToUpper(d.GetWordBeforeCursor()), true)...,
+	)
+}
+
+func (c *Client) help() {
+	fmt.Println(`Usage:
+        ` +
+		// ! UNSUPPORTED connect <host:port>   connects to another node specified by host:port
+		// ! UNSUPPORTED auth                  prompts for username and password
+		`pretty                toggles pretty print for the json format
+        use <db_name>         sets current database
+        format <format>       specifies the format of the server responses: json, csv, or column
+        precision <format>    specifies the format of the timestamp: h, m, s, ms, u or ns
+        history               displays command history
+        settings              outputs the current settings for the shell
+        clear                 clears settings such as database or retention policy.  run 'clear' for help
+        exit/quit/ctrl+d      quits the influx shell
+
+        show databases        show database names
+        show series           show series information
+        show measurements     show measurement information
+        show tag keys         show tag key information
+        show field keys       show field key information
+
+        A full list of influxql commands can be found at:
+        https://docs.influxdata.com/influxdb/latest/query_language/spec/`)
+	// ? Maybe add keybindings information like <CTRL+L> clearing the screen, <CTRL+D> exitting, etc
+}
+
+func (c *Client) settings() {
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 1, 1, ' ', 0)
+	fmt.Fprintln(w, "Setting\tValue")
+	fmt.Fprintln(w, "--------\t--------")
+	// TODO: No supported way to access or change host & port in a REPL session
```

Comment:
Is this something that might get fixed in a future PR? If so lets make an issue, if not lets take this commented code out (should probably remove the the commented out code either way)